docs/docs/reference/other-new-features/export.md

```diff
@@ -60,8 +60,7 @@ A member is _eligible_ if all of the following holds:
    a base class of the class containing the export clause.
  - it is accessible at the export clause,
  - it is not a constructor, nor the (synthetic) class part of an object,
- - it is a given instance (or an old-style `implicit` value)
-   if and only if the export is tagged with `given`.
+ - it is a given instance (declared with `given`) if and only if the export is from a _given selector_.
```

Comment:
this change comes from observation that implicit forwarders are only generated from wildcard selectors and not given selectors